app/js/components/labOrderEntry/DraftOrder/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+// import shortid from 'shortid';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import DraftList from './DraftList';
+
+export class LabDraftOrder extends React.Component {
+  render() {
+    const { draftLabOrders, disableCancelButton, disableSaveButton } = this.props;
+    return (
+      <div className="draft-spacing draft-lab-layout">
+        <h5 className="h5-draft-header">
+          Unsaved Draft Orders ({draftLabOrders.length})
+        </h5>
+        <div className="table-container">
+          <ul className="draft-list-container">
+            <DraftList
+              draftOrders={draftLabOrders}
+            />
+          </ul>
+        </div>
+        <br />
+        <input
+          type="button"
+          onClick={() => {}}
+          className="button cancel modified-btn"
+          value={draftLabOrders.length > 1 ? "Discard All" : "Discard"}
+          disabled={disableCancelButton}
+        />
+        <input
+          type="submit"
+          onClick={() => {}}
```

Comment:
What's this for?

Reply:
That's for the person going to implement the add feature

Reply:
Awesome thanks